sources/assets/web/html/custombar.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../css/custombar.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+</body>
```

Comment:
Instead of generating the page content, maybe you could do most stuff there fully statically? On some pages WebCord must generate the content (like *Settings*), but in places that aren't meant to be localized and can be safely defined, you should be able to add some tags so they don't have to be generated with JS.